lib/Service/NotesService.php

```diff
@@ -169,19 +171,28 @@ private static function gatherNoteFiles(Folder $folder, string $categoryPrefix =
 	/**
 	 * test if file is a note
 	 */
-	private static function isNote(FileInfo $file) : bool {
+    public function isNote(FileInfo $file, string $userId) : bool {
```

Comment:
Indentation is not correct (see lint) and public is not needed:
```suggestion
	private function isNote(FileInfo $file, string $userId) : bool {
```

Reply:
my commit correcting linting error and your review collided. Fully agree to make this function private